src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -251,5 +251,18 @@ public void unassignDisputes() {
 		logger.debug("UnassignDisputes called");
 		disputeService.unassignDisputes();
 	}
+	
+	/**
+	 * GET endpoint that retrieves the disputes filtered by JJ assigned
+	 *
+	 * @param jjAssigned
+	 * @return list of all disputes filtered by JJ assigned
+	 */
+	@GetMapping("/disputes/jj")
+	@Operation(summary = "Returns all disputes that have the specified JJ assigned.")
+	public ResponseEntity<List<Dispute>> getDisputesByJjAssigned(@RequestParam String jjAssigned) {		
+		List<Dispute> allJjAssignedDisputes = disputeService.getAllDisputesByJjAssigned(jjAssigned);
+		return new ResponseEntity<List<Dispute>>(allJjAssignedDisputes, HttpStatus.OK);
+	}
```

Comment:
So this is going to change in a future PR, right? To accept an optional jjAssigned param to return all JJDisputes, optionally filtered by assignment.

Reply:
That's right. We need to change this in a future PR to use the new JJDispute model to return all JJDisputes, optionally filtered by either jjGroupAssignedTo or jjAssignedTo fields.

Reply:
Perfect